src/mokutil.c

```diff
@@ -1873,11 +1883,13 @@ main (int argc, char *argv[])
 			{"ca-check",           no_argument,       0, 0  },
 			{"ignore-keyring",     no_argument,       0, 0  },
 			{"version",            no_argument,       0, 'v'},
+			{"verbose-listing",    no_argument,       0, 0},
```

Comment:
Nitpicking. Please add two spaces to the second `0` to align the brace.
Also, please add this option to the help message.